cmake/Sanitizers.cmake

```diff
@@ -0,0 +1,67 @@
+find_program(CLANG_APPLY_EXE NAMES "clang-apply-replacements" DOC "Path to clang-apply-replacements executable")
+find_program(CLANG_TIDY_EXE NAMES "clang-tidy" DOC "Path to clang-tidy executable")
+
+cmake_dependent_option(USE_CLANG_TIDY "Allow the codebase to use the clang-tidy sanitizer" ON "CLANG_TIDY_EXE" OFF)
+
+if(CMAKE_GENERATOR MATCHES "[Mm][Aa][Kk][Ee][Ff][Ii][Ll][Ee]|[Nn][Ii][Nn][Jj][Aa]")
```

Comment:
I think this is set to a constant string, so the silliness around case sensitivity is unnecessary...

Reply:
I aM rEmInDeD oF wHeN iT wAs KeWl To CaPiTaLiZe LiKe tHiS. :trollface: